src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "CinemaHalls")
```

Comment:
```suggestion
@Table(name = "cinema_halls")
```
Read about table naming convention